app/views/welcome/contact.html.erb

```diff
@@ -1,8 +1,27 @@
-<div class="row">
-  <div class="col-sm-4 content-container">
-    <iframe src="https://www.google.com/maps/d/u/0/embed?mid=1Mzqwzw1l67n9pskWjsMdyERVkCY" width="100%" height="480"></iframe>
+<div class="row contact-page">
+  <div class="col-sm-6">
+    <div class="text-container contact">
+      <%= render partial: 'welcome/address'%>
+    </div>
+    <div class="content-container">
+      <iframe src="https://www.google.com/maps/d/u/0/embed?mid=1Mzqwzw1l67n9pskWjsMdyERVkCY"  height="480"></iframe>
```

Comment:
is this the map that google maps tells you to embed? You shouldn't be using iframes.